doc/source/drivers/vector/miramon.rst

```diff
@@ -0,0 +1,370 @@
+
+- If the output is a **folder**, it will contain all the translated layers with the original name in the origin dataset.
+
+  - In this case a *.mmm* file will be created referencing all layers in the origin dataset to make an
+    easy open of the dataset using the MiraMon software.
+  - In this case, please specify the MiraMon file output format name using the -f option (**-f MiraMonVector**).
+
+- If it the output is a **file** with extension all the translated layers in the origin dataset will be created with the specified name.
+  Use this option only when you know that there is only one layer with one feature type in the origin dataset.
+
+When translating from a MiraMon vector format, the MiraMon vector driver input needs a file with one of the
+described extensions: *.pnt*, *.arc* or *.pol*. The extension *.nod* is not valid for translation.
+
+The attributes of the MiraMon feature are stored in an associated *.dbf*.
+If a classical DBF IV table could not be used (too many fields or records,
+large text fields, etc) a file type called extended DBF is used.
+This is an improvement of dBASE IV DBF files. The specification of this format can be found in `this file
+<https://www.miramon.cat/new_note/eng/notes/DBF_estesa.pdf>`__.
+
+Note that extended *.dbf* files cannot be opened with Excel or
+other typical programs. If the complete MiraMon Professional software
+is not installed on the computer, the free and standalone
+MiraD application can be downloaded from
+`this page <https://www.miramon.cat/USA/Prod-MiraD.htm>`__ to open them.
+
+Field sizes
+-----------
+
+The driver knows to auto-extend string and integer fields to
+dynamically accommodate for the length of the data to be inserted.
+
+Size Issues
+-----------
+
+Geometry: The MiraMon vector format explicitly uses 32-bit offsets in the 1.1 version
+and 64-bit offsets in the 2.0 version. It is better to produce 1.1 version files if 2.0
+version is not really necessary than always use 2.0 version. Version 1.x files are smaller.
+
+Attributes: The dbf format does not have any offsets in it, so it can be
+arbitrarily large.
+
+Open options
+------------
+
+The following open options are available.
+
+-  .. oo:: Height
+      :choices: First, Lowest, Highest
+
+      Sets which of the possible heights for each vertex is read: 
+      the *first*, the *lowest* or the *highest* one. It only applies to
+      MiraMon multi-height layers, where the same X,Y vertex can have more than one Z.
+
+-  .. oo:: iMultiRecord
+      :choices: 1, 2, ..., Last, JSON
+
+      In case of fields of type List, if the output driver cannot support them,
+      user can select which one wants to keep: *iMultiRecord=1* for first, *iMultiRecord=2* for second, etc
+      and *iMultiRecord=last* for the last element of the list.
+      *iMultiRecord=JSON* option converts the list in a single value in JSON format.
+      If not specified, all elements of the list will be translated by default.
```

Comment:
I tryed to program this hint you gave in an e-mail ([gdal-dev] Fields of type List to simple fields with N times the feature?):
Actually, an alternative would be that you type your list fields with type=OFTString and subtype=OFSTJSon, and emit a serialized JSON array (``[1,2]``). This is probably more in the spirit of nowadays uses, and that should make all output drivers happy (at worse they won't understand the JSON hint)  

Should I write "converts the list in a single string in a JSON array format." instead of "converts the list in a single value in JSON format."

Reply:
> Should I write "converts the list in a single string in a JSON array format."

yes.
But my question was specifically on what "If not specified, all elements of the list will be translated by default." meant exactly. Is it another format that the iMultiRecord=JSON one ? Or is JSON the default format ?

Reply:
It's the original type. If it's an integer then an integer, if it's a string, a string, if its a stringlist, a stringlist...
MiraMon supports StringList, IntegerList, etc... 

It returns to GDAL the type that this value is (do not modify the type). So it does nothing special. so, if nothing is specified in iMultiRecord the Field will be of that type.
Better now? 

Feel free to propose a better sentence :)

Reply:
maybe "If not specified, all elements of the list will be translated by default as a OGR list field type"

Reply:
Done!